.github/actions/clp-core-build/action.yaml

```diff
@@ -11,27 +11,25 @@ runs:
   using: "composite"
   steps:
 
+
     - name: Download submodules
       run: ./tools/scripts/deps-download/download-all.sh
-      working-directory: $GITHUB_WORKSPACE/components/core
+      working-directory: ./components/core
```

Comment:
How come $GITHUB_WORKSPACE is unnecessary?

Reply:
This composite action executes inside the clp-core-dependencies container and there is a whole mixup with contexts when the action runs inside a container. $GITHUB_WORKSPACE points to a path on the host VM which is mounted to the running container. All the build binaries need to be output to $GITHUB_WORKSPACE  in order for the upload-artifact action to acess them because that action executes natviely on the host VM. The repository source code however does not live at $GITHUB_WORKSPACE so that causes an error, it is mounted at the entrypoint path so we have to use "./"

Reply:
We could have used `${{github.workspace}}/components/core` which maps to the current workspace inside the container,  but I felt that would just cause more confusion as `${{github.workspace}}` and `$GITHUB_WORKSPACE` point to diffent paths

Reply:
Gotcha, gotcha. Thanks for explaining.